java/23c-jdbc-boolean/src/main/java/com/oracle/dev/jdbc/QueryWithBooleanDataTypeColumn.java

```diff
@@ -0,0 +1,71 @@
+/*
+  Copyright (c) 2024, Oracle and/or its affiliates.
+
+  This software is dual-licensed to you under the Universal Permissive License
+  (UPL) 1.0 as shown at https://oss.oracle.com/licenses/upl or Apache License
+  2.0 as shown at http://www.apache.org/licenses/LICENSE-2.0. You may choose
+  either license.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+package com.oracle.dev.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import oracle.jdbc.pool.OracleDataSource;
+
+public class QueryWithBooleanDataTypeColumn {
+
+	public static void main(String[] args) {
+
+		OracleDataSource ods = null;
+		Connection conn = null;
+
+		try {
+			
+			ods = new OracleDataSource();
+			
+			// jdbc:oracle:thin@[hostname]:[port]/[DB service/name]
+			ods.setURL("jdbc:oracle:thin:@localhost:1521/FREEPDB1");			
+			ods.setUser("[Username]");
+		    ods.setPassword("[Password]");	
+						
+			conn = ods.getConnection();
+			
+			PreparedStatement stmt = conn.prepareStatement("SELECT * FROM HQ_EMPLOYEE WHERE ACTIVE");						
```

Comment:
Where is HQ_EMPLOYEE defined?

Reply:
Add this as a code comment:
CREATE TABLE HQ_EMPLOYEE
(
  "EMP_ID"   NUMBER NOT NULL ENABLE,
  "NAME"    VARCHAR2(20 BYTE) DEFAULT NULL,
  "ROLE"    VARCHAR2(20 BYTE) DEFAULT NULL,
  "ACTIVE" BOOLEAN DEFAULT NULL,
  PRIMARY KEY ("EMP_ID")
);